sentry-failure/src/lib.rs

```diff
@@ -49,6 +50,33 @@ fn fail_typename<F: Fail + ?Sized>(f: &F) -> (Option<String>, String) {
     }
 }
 
+#[derive(Default)]
+pub struct FailureIntegration;
```

Comment:
Can we give all public types and functions a doc comment, and potentially a doc test? Also applies to the `sentry-panic` crate. We should specifically mention that this is a default integration.